README.md

```diff
@@ -76,6 +76,45 @@ JDA jda = new JDABuilder("token").build();
 You can change this to use `AccountType.CLIENT`, however you will be risking account termination.
 Use `new JDABuilder(AccountType)` to change to a different account type.
 
+### Configuration
+
+Both the `JDABuilder` and the `DefaultShardManagerBuilder` allow a set of configurations to improve the experience.
+
+**Example**:
+
+```java
+public static void main(String[] args) {
+    JDABuilder builder = new JDABuilder(args[0]);
+    
+    // Disable parts of the cache
+    builder.setDisabledCacheFlags(EnumSet.of(CacheFlag.ACTIVITY, CacheFlag.VOICE_STATE));
+    // Enable the bulk delete event
+    builder.setBulkDeleteSplittingEnabled(false);
+    // Disable compression (not recommended)
+    builder.setCompression(Compression.NONE);
+    // Set activity (like "playing Something")
+    builder.setActivity(Activity.watching("TV"));
+    
+    builder.build();
+}
+```
+
+> See [JDABuilder](https://ci.dv8tion.net/job/JDA4-Beta/javadoc/net/dv8tion/jda/api/JDABuilder.html)
+  and [DefaultShardManagerBuilder](https://ci.dv8tion.net/job/JDA4-Beta/javadoc/net/dv8tion/jda/api/sharding/DefaultShardManagerBuilder.html)
+
+### Listening to Events
+
+The event system in JDA is configured through a hierarchy of classes/interfaces.
+We offer two implementations for the `IEventManager`:
```

Comment:
We should probably mention that you can implement and provide your own implementation of `IEventManager`